README.rst

```diff
@@ -47,10 +47,11 @@ module ``texasbbq`` will then provides a command-line interface for running the
 tests (see blow).
 
 The main entry point is a single script, ``texasbbq.py``, which is used to
-drive integration testing. This script will run on at least Python 2.7 and
-3.7 and has zero third-party dependencies. Hence it will probably run on a
-large variety of different CI systems and platforms. It provides a pure Python
-interface to ``miniconda`` without using a shell language.
+drive integration testing. This script will run on at least Python 2.7 and 3.7
```

Comment:
Is there a python 2 version of conda 4.9?

Reply:
Probably not, no, but that doesn't matter as Python 2.7 is only needed to bootstrap a miniconda which will include the correct version of `conda`.

Reply:
Also, we always use `subprocess` to execute `conda` rather than importing it, which is another layer of isolation.

Reply:
ok, I don't think it's clear that it downloads its dependencies, xref https://github.com/numba/texasbbq/pull/22/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR68

Reply:
Hmmm, yes, this is somewhat unusual. Do you have any thoughts on how best to specify that?

Reply:
Shall I perhaps add another sentence to the end of this paragraph to clarify that `miniconda` and `conda` will be downloaded?

Reply:
> Shall I perhaps add another sentence to the end of this paragraph to clarify that `miniconda` and `conda` will be downloaded?

probably a good idea

Reply:
done